UXWTypography.js

```diff
@@ -1,94 +1,181 @@
-            <td className="px-4 py-2">1.75</td>
-            <td className="px-4 py-2">27px</td>
-            <td className="px-4 py-2 font-bold" style={{fontSize: "1.75em", lineHeight: 1}}>Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Regular</td>
-            <td className="px-4 py-2">1.5em</td>
-            <td className="px-4 py-2">24px</td>
-            <td className="px-4 py-2 text-2xl">Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Regular</td>
-            <td className="px-4 py-2">1.313em</td>
-            <td className="px-4 py-2">21px</td>
-            <td className="px-4 py-2" style={{fontSize: "1.313em"}}>Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Bold</td>
-            <td className="px-4 py-2">1em</td>
-            <td className="px-4 py-2">16px</td>
-            <td className="px-4 py-2 font-bold">Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Regular</td>
-            <td className="px-4 py-2">1em</td>
-            <td className="px-4 py-2">16px</td>
-            <td className="px-4 py-2">Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Regular</td>
-            <td className="px-4 py-2">0.875em</td>
-            <td className="px-4 py-2">14px</td>
-            <td className="px-4 py-2 text-sm">Design meets magic</td>
-          </tr>
-          <tr>
-            <td className="px-4 py-2">Regular</td>
-            <td className="px-4 py-2">0.75em</td>
-            <td className="px-4 py-2">12px</td>
-            <td className="px-4 py-2 uppercase text-xs">Design meets magic</td>
-          </tr>
-        </tbody>
-      </table>
-      <h2 id="typography--guidelines" className="text-2xl pb-4 pt-16">Guidelines</h2>
-      <h3 className="text-xl pb-4 pt-8">Leading</h3>
-      <p>Ideally, body type should have a line-height of at least 125% of the font size. For example, a paragraph of font size 14 with a line-height of 24.
-      </p>
-      <h3 className="text-xl pb-4 pt-8">Character length</h3>
-      <p>Limit line length of body text between 50-75 characters for better readability.
-      </p>
-      <h3 className="text-xl pb-4 pt-8">Accessibility</h3>
-      <p>Accessibility comes first. Type colors should meet WCAG 2.0 standard for contrast.
-        ↳ Color contrast guidelines
-      </p>
-      <h2 id="typography--usage" className="text-2xl pb-4 pt-16">Usage</h2>
-      <p>Example</p>
-      <h2 id="typography--css" className="text-2xl pb-4 pt-16">CSS</h2>
+      <UXWSubsection anchor="typography--font" title="Font">
+        <p className="max-w-prose">Inspired by the typeface used in road signage all around America, Overpass represents the UX Wizards’ ever expanding reach.
+        </p>
+        <a href="https://fonts.google.com/specimen/overpass">↳ Download Overpass</a>

```

Comment:
This link is broken. I think you need to change `overpass` to `Overpass` in the url